wordpress-mobile/WordPress-iOS
# Add react-native-slider library

Fixes #

I added within `gutenberg-mobile` new library `react-native-slider` which is going to replace `Slider` from `react-native` which will be deprecated soon and what's more have a bug on ios ~13.

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/18806
Ref to gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1620

To test:

I was testing it in WordPress-Android, WordPress-iOS and also within a playground.

1. Add `Spacer` block on mobile
2. Play with `Slider`
  * I suggest you also to set small range eg. minValue=0, maxValue=20 in [RangeCell](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/mobile/bottom-sheet/range-cell.native.js#L181-L182) to check if Slider is not jumping



To test:

PR submission checklist:
